### ISSUE TITLE
Newest GMOD update causes the loading to the title screen to soft-lock

### ISSUE THREAD

**Issue by MagentaIris** [Facepunch/garrysmod-issues#3490]
Unknown cause, I have cleaned garrysmod twice so far including factory resets via the .BAT

The game seems to get to the blue screen with the LOADING... at the bottom right hand corner but never continues past this point, I also think -tools mode is broken but i don't have enough evidence of that yet due to not even being able to load into the game itself.

**Comment by robotboy655** [comment]
Does the chromium branch work? Any other Source engine games? Any pop up messages in background or w/e?

This shouldn't be caused by this update since nothing about the starting process has changed, quite literally in years probably.

**Comment by MagentaIris** [comment]
I have tried every single Beta branch there is, same issue applies.

I can alt-tab with windowed or even noborder, I have tried launching the game without these command lines and it still acts the same, the game is in a sort of hibernated state, as if it's just stopped loading anything, its running and it doesn't white screen to "Has stopped responding"

There are no error messages of any kind. HL2: Episode 2 runs perfectly fine.

TL;DR - Chromium has the same issue, Episode 2 works fine, No messages of any kind.

Heres the screen i'm talking about just to help with clarity.

![image](https://user-images.githubusercontent.com/4176672/40751463-2e9f6e4e-6463-11e8-855d-582c9bc713db.png)

**Comment by robotboy655** [comment]
Send a dump file from task manager then to ```robotboy655@facepunchstudios.com``` or something then.

**Comment by MagentaIris** [comment]
Now this is new, For some reason after launching a normal source engine game, Garrysmod now launches properly and works. I'll keep this post updated incase anything else happens regarding the soft lock.

**Comment by MagentaIris** [comment]
It seems that Nvidias "Shadow Play" plugin/option is what causes gmod to stop loading, i decided to disable it completely to get to the title screen, load up a single player game on gm_construct, and mid-loading re-enable Shadow Play, the loading bar froze while the game didn't crash at all, causing another soft lock, The issue seems to be Shadow play.

**Comment by robotboy655** [comment]
Doesn't that have some sort of overlay? GMod is known to have problems with overlays.

**Comment by MagentaIris** [comment]
That's correct, it does have a type of overlay, it appears at the top right of the screen upon initializing, that's most likely the cause since thats when it starts to freak out.

**Comment by robotboy655** [comment]
Going to close this as stale. If the issue happens again, open a new report.
